packages/next/src/server/require.ts

```diff
@@ -57,6 +57,9 @@ export function getMaybePagePath(
     page = denormalizePagePath(normalizePagePath(page))
   } catch (err) {
     console.error(err)
+    if (err instanceof NormalizeError) {
+      throw err
+    }
```

Comment:
This feels like a really nice way to handle this! Raising `PageNotFoundError` when the error came from the normalize/denormalize process seems strange. Allowing that error to bubble up so it can be properly handled is great!